fastembed/models.json

```diff
@@ -0,0 +1,113 @@
+[
+    {
+        "model": "BAAI/bge-base-en",
+        "dim": 768,
+        "description": "Base English model",
+        "size_in_GB": 0.5,
+        "hf_sources": [],
+        "gcs_sources": [
```

Comment:
Do NOT limit to GCS. 

Have the complete URL here. 

So when writing code for this now — have everything against a URL, without assuming the url structure of GCS. 

With that in mind, perhaps the key should be: "compressed_url_sources" ?